app/views/rooms/edit.html.erb

```diff
@@ -20,3 +20,17 @@
     <%= render "furnitures/new", furniture: new_furniture %>
   <%- end %>
 <% end %>
+
+
+<%- if policy(room).destroy? && room.persisted? && !room.entrance? %>
+  <div class="m-5">
+    <h2>Remove this Section?</h2>
+    <%- data = {} %>
+    <%- if !room.gizmos.reload.empty? %>
+      <p>Warning! Deleting this Section will also delete all Gizmos and their related data. Please be very sure you want to do this! We recommend making the Section "internal" instead.</p>
```

Comment:
Ideally these strings would be I18n-ed, but I don't think that's a blocker for merging this.

Good call on this warning and on having a `confirm` on the delete button.

Reply:
True, ugh